src/test_application/activescenes/scenarios_vehicletest.cpp

```diff
@@ -259,6 +283,25 @@ void VehicleTest::setup_scene(CommonTestScene &rScene, osp::PkgId pkg)
        { fueltank, Matrix4::translation({0, 0, 0}) },
        { engine,   Matrix4::translation({0, 0, -3}) },
     });
+
+
+    namespace ports_magicrocket = adera::ports_magicrocket;
+    namespace ports_userctrl = adera::ports_userctrl;
+
+    auto const [ pitch, yaw, roll, throttle ] = rBuilder.create_nodes<4>(gc_ntNumber);
```

Comment:
i recommend making the habit to always do `auto const& [...]`, so that in case the item being returned has an expensive copy-constructor, you're not paying that cost.